src/main/java/org/jenkinsci/test/acceptance/po/WizardLogin.java

```diff
@@ -0,0 +1,72 @@
+package org.jenkinsci.test.acceptance.po;
+
+import static org.hamcrest.Matchers.not;
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.jenkinsci.test.acceptance.controller.JenkinsController;
+import org.jenkinsci.test.acceptance.controller.LocalController;
+import org.junit.Assert;
+
+import com.google.inject.Inject;
+
+/**
+ * Page object for Wizard Login page.
+ *
+ */
+public class WizardLogin extends PageObject {
+    @Inject
+    JenkinsController controller;
+
+
+    private Control cPassword = control("/j_password");
+    private Control cLogin = control("/Continue");
+
+    public WizardLogin(Jenkins jenkins) {
+        super(jenkins.injector, jenkins.url(""));
+    }
+
+    /**
+     * Login assuming "path" form element availability.<br>
+     * Paths are usually available when using the default Jenkins controller.<br>
+     * (Available thanks to pre-installed form-element-path plugin.)
+     */
+    public WizardLogin doLogin(String password) {
+        cPassword.set(password);
+        cLogin.click();
+        return this;
+    }
+
+    /**
+     * Login assuming "path" form element unavailability.<br>
+     * Paths are usually unavailable when using the "existing" Jenkins controller.<br>
+     * (Unavailable despite pre-installed form-element-path plugin.)
+     */
+    public WizardLogin doLoginDespiteNoPaths(String password) {
+        driver.findElement(by.name("j_password")).sendKeys(password);
+        clickButton("Continue");
+        return this;
+    }
+
+    public WizardLogin doSuccessfulLogin(String password) {
+        this.doLoginDespiteNoPaths(password);
+        Assert.assertThat(driver, not(hasContent("The password entered is incorrect")));
+        return this;
+    }
+
+    /**
+     * Gets the generated password from the file on disk
+     * 
+     * @return the read password
+     * @throws IOException if there is an IO error 
+     */
+    public String getPassword() throws IOException {
+        File passwordFile = new File(((LocalController) controller).getJenkinsHome(), "secrets/initialAdminPassword");
```

Comment:
Yes and no... this test will only run if the controller is `LocalController`, but potentially this could happen if we remove the restriction. Will change it.